src/build_options.rs

```diff
@@ -430,7 +434,7 @@ pub fn find_bridge(cargo_metadata: &Metadata, bridge: Option<&str>) -> Result<Br
         if !pyo3_node.features.contains(&"extension-module".to_string()) {
             let version = cargo_metadata[&pyo3_node.id].version.to_string();
             println!(
```

Comment:
Should this be `eprintln!` ? Maybe there's value in a `warn!` macro:

```rust
macro_rules! warn {
    ($msg:expr, $($args:tt)+) => {
        eprintln!(concat!("⚠️ Warning: ", $msg), $($args)+)
    };
}
```

Reply:
Further cleanups can be done separately.